src/Process/3dTilesProcessing.js

```diff
@@ -257,24 +257,36 @@ export function pre3dTilesUpdate(context, layer) {
 
 const boundingVolumeBox = new THREE.Box3();
 const boundingVolumeSphere = new THREE.Sphere();
-export function computeNodeSSE(camera, node) {
+export function computeNodeSSE(context, camera, node) {
     node.distance = 0;
     if (node.boundingVolume.region) {
         boundingVolumeBox.copy(node.boundingVolume.region.box3D);
         boundingVolumeBox.applyMatrix4(node.boundingVolume.region.matrixWorld);
         node.distance = boundingVolumeBox.distanceToPoint(camera.camera3D.position);
+        // update distance min/max
+        const s = node.boundingVolume.region.box3D.getSize();
+        const maxComponent = Math.max(s.x, Math.max(s.y, s.z));
+        context.distance.min = Math.min(context.distance.min, node.distance - maxComponent);
```

Comment:
the calculations of min and max can be done, only once, after the calculation of the `boudingVolume`

The name of the function must also take into account this new calculation

Reply:
in line 293